data_steward/analytics/cdr_ops/raw_rdr_export_qc.py

```diff
@@ -702,6 +702,30 @@
 query = tpl.render(new_rdr=new_rdr, project_id=project_id)
 execute(client, query)
 
+# # Check the expectations of survey_conduct - survey list
+#
+# Confirm that all expected surveys have records in survey_conduct. Check ignores snap surveys because these surveys are not expected in any release.
+#
+# Generally the list of surveys should increase from one export to the next.
+#
+# Investigate any surveys that were available in the previous export but not in the current export. 
+# Also make sure that any new expected surveys are listed in the current rdr.
+
+tpl = JINJA_ENV.from_string('''
+SELECT
+  prev.survey_source_value AS survey_in_previous_rdr,
+  curr.survey_source_value AS survey_in_current_rdr
+FROM (SELECT DISTINCT survey_source_value FROM `{{project_id}}.{{new_rdr}}.survey_conduct`) curr
+FULL OUTER JOIN (SELECT DISTINCT survey_source_value FROM `{{project_id}}.{{old_rdr}}.survey_conduct`) prev
+  USING (survey_source_value)
```

Comment:
It would be good to have counts here as well
```suggestion
  prev.survey_source_value AS survey_in_previous_rdr,
  prev.n AS survey_count_in_previous_rdr
  curr.survey_source_value AS survey_in_current_rdr,
  curr.n AS survey_count_in_curr_rdr
FROM (SELECT survey_source_value, COUNT(survey_source_value) n
FROM `{{project_id}}.{{new_rdr}}.survey_conduct`
GROUP BY survey_source_value) curr
FULL OUTER JOIN (SELECT survey_source_value, COUNT(survey_source_value)
FROM `{{project_id}}.{{old_rdr}}.survey_conduct`
GROUP BY survey_source_value) prev
  USING (survey_source_value)
```